### PR TITLE
Create sql

### PR DESCRIPTION
**Pull request by OlegKovalenko1984**
add sql

### CODE REVIEW

File: sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
+create table restoraunt_info(id serial,
+	                         r_name varchar(200) not null,
+							 delivery char(1) not null,
+							 link_to_site varchar(200),
+							 count_of_employers int,
+							 alco char(1),
```
Comment (Andriireva) [inline sql]: change type to boolean and change name to "is_alcohol_allowed"

File: sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
+create table restoraunt_info(id serial,
+	                         r_name varchar(200) not null,
```
Comment (Andriireva) [inline sql]: change to name it is not required to add prefix r_

File: sql [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+							 delivery char(1) not null,
```
Comment (Andriireva) [inline sql]: change name to "has_delivery" and change type to boolean

File: sql [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+							 link_to_site varchar(200),
+							 count_of_employers int,
+							 alco char(1),
+							 summer_platform char(1),
```
Comment (Andriireva) [inline sql]: change to "has_summer_platform" and change type to boolean

File: sql [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+							 CONSTRAINT pk PRIMARY KEY (id))							)
+
+insert into restoraunt_info(r_name,delivery,link_to_site,count_of_employers,
+							 alco,summer_platform ,time_of_work)
+values('Весёлый бородач','y','http://borodach.com',30,'y','y','8.00-20.00')
+
+
+create table menu(id serial,
+				   id_restoraunt int not null references restoraunt_info(id),
+				    m_name varchar(150) not null,
```
Comment (Andriireva) [inline sql]: remove prefixes it is not needed

File: sql [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+				    m_price numeric(15,2) not null,
```
Comment (Andriireva) [inline sql]: remove prefixes it is not needed

**Review by Andriireva — CHANGES_REQUESTED** [review]

File: sql (hunk 1 above)
Comment (OlegKovalenko1984) [reply sql]: Fixed

File: sql (hunk 2 above)
Comment (OlegKovalenko1984) [reply sql]: Fixed

File: sql (hunk 3 above)
Comment (OlegKovalenko1984) [reply sql]: Fixed

File: sql (hunk 4 above)
Comment (OlegKovalenko1984) [reply sql]: Fixed

File: sql (hunk 6 above)
Comment (OlegKovalenko1984) [reply sql]: Fixed

**Review by Andriireva — APPROVED** [review]

**Pull request merged**